content/en/blog/2021/wasm-progress/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Progress of Envoy and Istio WebAssembly Extensibility
+description: An update on Envoy and Istio WebAssembly based extensibility effort.
+publishdate: 2021-02-09
+attribution: "Pengyuan Bian (Google)"
+keywords: [wasm,extensibility,WebAssembly]
+---
+
+Istio introduced initial [WebAssembly-based extensibility](/blog/2020/wasm-announce/) support in the 1.5 release.
+Over the past year, Istio, Envoy, and Proxy-Wasm communities have continued the effort to make WebAssembly extensibility stable, reliable, and easy to adopt.
+In this blog post, we will walk through the updates to Wasm support in the Istio 1.9 release.
+
+## Proxy-Wasm (WebAssembly) support merged in upstream Envoy
+
+After our initial experimental support was available, we collected great feedback from community early adopters.
+Along with experience gained from development efforts on core Istio Wasm extensions, this feedback helped us mature and stabilize the runtime.
+These improvements unblocked merging Wasm support directly into Envoy, allowing it to become part of all official Envoy releases (as of the Istio 1.8 release).
+This is a significant milestone for Istio and Envoy Wasm efforts, since it indicates that:
+
+* The runtime is ready for wider adoption.
+* The programming ABI/API, extension configuration API, as well as runtime behavior are becoming stable.
+* Better community support moving forward.
+
+## Istio Wasm Extension Ecosystem
+
+As one of the early adopters of Envoy WebAssembly runtime, Istio extensibility working group gained a lot of experience via developing several first class extensions, such as [metadata exchange](/docs/reference/config/proxy_extensions/metadata_exchange/), [Prometheus stats](/docs/reference/config/proxy_extensions/stats/), and [attribute generation](/docs/reference/config/proxy_extensions/attributegen/).
```

Comment:
nit: do you want to say `Istio Extensions and Telemetry working group` here for consistency with the group names?

Reply:
Done.